### PR TITLE
Feature/farm 428 minor ui fixing

### PR DESCRIPTION
**Pull request by isaacmartinezimv**
[[FARM-428] Minor UI Fixing](https://amidodevelopment.atlassian.net/browse/FARM-428?atlOrigin=eyJpIjoiMGYzOTZhYWYxOThiNDcyODhmODdlNmVkZTZkMDdlNmQiLCJwIjoiaiJ9)

#### 📲 What

A few changes on UI's of Profit and Loss and MyPlot.

#### 🤔 Why
		
To match the designs.
	
#### 🛠 How
		
Using styles

#### ✅ Acceptance criteria

- [X] Design Review for UI with BA completed. 
- [X] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
- [ ] Showcase video / automation test
- [ ] Passing all tests
- [X] Rebased/merged with latest changes from development and re-tested
- [ ] Removed unsed comments. TODOs must have JIRA for future work.

#### Coding Standards Checklist
- [ ] unit tests 80% coverage on testable code (functions, methods, class)
- [X] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
- [ ] ui elements defined under lib/ui. 
- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
- [ ] Colors defined in lib/utils/colors.dart
- [ ] Dimensions (ie margins padding) defined in lib/utils/dimens.dart
- [ ] TextStyles defined in lib/utils/styles.dart

Recommended Style Guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

#### 🕵️‍♂️ How to test

Notes for QA

#### Reviewers

### CODE REVIEW

File: lib/ui/common/Dogtag.dart [hunk 1]
```diff
@@ -126,7 +131,7 @@ class DogTag extends StatelessWidget {
         Icon(
           viewModel.icon,
           size: style.iconSize,
-          color: Colors.black,
+          color: style.iconColor,
```
Comment (davidfranquet) [inline lib/ui/common/Dogtag.dart:127]: Thanks ! :)

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 2]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+      PlaygroundWidget(
+          title: 'RoundedButton Large',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.all(20.0),
+            height: 96,
+            child: RoundedButton(
+                viewModel: MockRoundedButtonViewModel.buildLarge(),
+                style: RoundedButtonStyle.largeRoundedButtonStyle()),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 3]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+      PlaygroundWidget(
+          title: 'RoundedButton Compact',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.all(20.0),
+            height: 64,
+            child: RoundedButton(
+                viewModel: MockRoundedButtonViewModel.buildCompact(),
+                style: RoundedButtonStyle.defaultStyle()),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 4]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+      PlaygroundWidget(
+          title: 'RoundedButton Compact Big',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.all(20.0),
+            height: 88,
+            child: RoundedButton(
+                viewModel: MockRoundedButtonViewModel.buildCompact(),
+                style: RoundedButtonStyle.bigRoundedButton()),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 5]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+      PlaygroundWidget(
+          title: 'DogTag Days',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.all(20.0),
+            height: 100,
+            child: DogTag(
+                viewModel: MockDogTagViewModel.buildWithText(),
+                style: DogTagStyle.compactStyle()),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 6]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+      PlaygroundWidget(
+          title: 'DogTag Positive',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.all(20.0),
+            height: 1000,
+            child: DogTag(
+                viewModel: MockDogTagViewModel.buildWithPositiveNumber(),
+                style: DogTagStyle.defaultStyle()),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 7]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+      PlaygroundWidget(
+          title: 'DogTag Negative',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.all(20.0),
+            height: 100,
+            child: DogTag(
+                viewModel: MockDogTagViewModel.buildWithNegativeNumber(),
+                style: DogTagStyle.negativeStyle()),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 8]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+
+      PlaygroundWidget(
+          title: 'Simple Action Sheet Item',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.symmetric(vertical: 20.0),
+            child: ActionSheetListItem(
+                viewModel:
+                    MockActionSheetViewModel.buildStandard().actions.first),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_atom_datasource.dart [hunk 9]
```diff
@@ -17,33 +18,112 @@ class PlayGroundAtomDataSource implements PlaygroundDataSource {
...
+      PlaygroundWidget(
+          title: 'Action Sheet Item with Checkbox',
+          child: Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.symmetric(vertical: 20.0),
+            child: ActionSheetListItem(
+                viewModel:
+                    MockActionSheetViewModel.buildWithCheckBox().actions.first),
+          )),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_atom_datasource.dart]: Add missing comma and do format code

File: lib/ui/playground/data/playground_tasks_datasource.dart [hunk 10]
```diff
@@ -11,59 +12,75 @@ import 'package:farmsmart_flutter/ui/profitloss/mockRepositoryTryout/MockTransac
...
+                        style: ActionSheetStyle.defaultStyle()),
+                  )),
+              PlaygroundWidget(
+                  title: 'FARM-355 Generic Action Sheet - With checkBox',
+                  child: Container(
+                      height: 350,
+                      child: ActionSheet(
+                          viewModel:
+                              MockActionSheetViewModel.buildWithCheckBox(),
+                          style: ActionSheetStyle.defaultStyle()))),
```
Comment (davidfranquet) [inline lib/ui/playground/data/playground_tasks_datasource.dart:81]: Add missing commas and do format code

**Review by davidfranquet — DISMISSED** [review]

File: lib/ui/common/Dogtag.dart [hunk 11]
```diff
@@ -21,37 +21,40 @@ class DogTagStyle {
...
       {this.backgroundColor,
       this.borderRadius,
       this.edgePadding,
       this.titleTextStyle,
       this.maxLines,
       this.iconSize,
+      this.iconColor,
       this.spacing});
 
   factory DogTagStyle.defaultStyle() {
```
Comment (LeeMatthewHiggins) [inline lib/ui/common/Dogtag.dart]: this is not in the coding style

File: lib/ui/common/Dogtag.dart [hunk 12]
```diff
@@ -21,37 +21,40 @@ class DogTagStyle {
...
-            color: Color(0xff25df0c)),
+            fontWeight: FontWeight.w500,
+            color: Color(0xff21c400)),
         maxLines: 1,
         iconSize: 8,
+        iconColor: Colors.black,
         spacing: 5.5);
   }
 
   factory DogTagStyle.negativeStyle() {
```
Comment (LeeMatthewHiggins) [inline lib/ui/common/Dogtag.dart]: We should only have 1 private default style in the widget. All. Other styles should be outside the widget at the point of use.

File: lib/ui/myplot/PlotListItem.dart [hunk 13]
```diff
@@ -49,13 +50,24 @@ abstract class PlotListItemStyle {
...
+      this.imageSize,
+      this.detailTextBackgroundColor,
+      this.detailLineSpace,
+      this.headingLineSpace,
+      this.overlayColor,
+      this.imageLineSpace,
+      this.maxLineText);
 }
 
 class _defaultStyle implements PlotListItemStyle {
```
Comment (LeeMatthewHiggins) [inline lib/ui/myplot/PlotListItem.dart]: _DefaultStyle     (caps)

File: lib/ui/myplot/PlotListItem.dart [hunk 14]
```diff
@@ -64,20 +76,26 @@ class _defaultStyle implements PlotListItemStyle {
...
 
-  final BorderRadius detailTextBorderRadius = const BorderRadius.all(Radius.circular(20.0));
+  final BorderRadius detailTextBorderRadius =
+      const BorderRadius.all(Radius.circular(20.0));
 
   final double elevation = 0.0;
   final double imageSize = 80.0;
-  final double headingLineSpace = 12.5;
+
+  //FIXME: CHeckThat!
```
Comment (LeeMatthewHiggins) [inline lib/ui/myplot/PlotListItem.dart]: can we check now? :)

File: lib/ui/profitloss/mockRepositoryTryout/MockTransactionRepository.dart [hunk 15]
```diff
@@ -47,9 +47,18 @@ MockString _mockDetailText = MockString(library: [
   "A Bit Longer day 9000"
 ]);
 
-MockString _mockPositiveCurrencyText = MockString(library: ["255", "0", "2", "300", "450", "25"]);
+MockString _mockCurrencyText = MockString(library: [
+  "Eur",
+  "Usd",
+  "KSh",
```
Comment (LeeMatthewHiggins) [inline lib/ui/profitloss/mockRepositoryTryout/MockTransactionRepository.dart:53]: 👍

**Review by LeeMatthewHiggins — CHANGES_REQUESTED** [review]

File: lib/ui/myplot/PlotListItem.dart (hunk 14 above)
Comment (isaacmartinezimv) [reply lib/ui/myplot/PlotListItem.dart]: Yeah was fixed yesterday but i forgot to delete the FIXME

File: lib/ui/common/DogTagStyles.dart [hunk 16]
```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:farmsmart_flutter/ui/common/Dogtag.dart';
+
+class DogTagStyles {
```
Comment (LeeMatthewHiggins) [inline lib/ui/common/DogTagStyles.dart:5]: 👍

**Review by LeeMatthewHiggins — APPROVED** [review]
Good Work

**Review by davidfranquet — APPROVED** [review]

**Pull request merged**